ledger/acctonline.go

```diff
@@ -619,19 +619,7 @@ func (ao *onlineAccounts) onlineTotals(rnd basics.Round) (basics.MicroAlgos, pro
 // LookupOnlineAccountData returns the online account data for a given address at a given round.
 func (ao *onlineAccounts) LookupOnlineAccountData(rnd basics.Round, addr basics.Address) (data basics.OnlineAccountData, err error) {
 	oad, err := ao.lookupOnlineAccountData(rnd, addr)
```

Comment:
We might as well eliminate the named return values, and return the inner call directly.

```suggestion
func (ao *onlineAccounts) LookupOnlineAccountData(rnd basics.Round, addr basics.Address) (basics.OnlineAccountData, error) {
	return ao.lookupOnlineAccountData(rnd, addr)
```

Reply:
applied and resolved conflicts